### PR TITLE
Fix typo in distrubuted-tracing-concepts.md

### PR DESCRIPTION
**Pull request by CraigComeOnThisNameCantBeTaken**
## Summary

Describe your changes here.
Small typo fix

Fixes #Issue_Number (if available)

### CODE REVIEW

File: docs/core/diagnostics/distributed-tracing-concepts.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ or a library provided by a third-party telemetry or APM vendor. Alternately deve
 their own custom Activity telemetry collection by using
 <xref:System.Diagnostics.ActivityListener?displayProperty=nameWithType> or
 <xref:System.Diagnostics.DiagnosticListener?displayProperty=nameWithType>. ActivityListener
-supports observing any Activity regardless whether the developer has any a priori knowledge about it.
+supports observing any Activity regardless whether the developer has any a prior knowledge about it.
```
Comment (BillWagner) [inline docs/core/diagnostics/distributed-tracing-concepts.md:104]: The original "a priori" was correct, but a non-English phrase. For the new spelling to be correct, I'd remove the "a" as well:

```suggestion
supports observing any Activity regardless whether the developer has any prior knowledge about it.
```

**Review by BillWagner — COMMENTED** [review]
Thanks for making this change @CraigComeOnThisNameCantBeTaken (love the GitHub handle 😄 ).  

I'll merge this after you accept my one suggestion.

Thanks again.

**Comment by CraigComeOnThisNameCantBeTaken** [comment]
> Thanks for making this change @CraigComeOnThisNameCantBeTaken (love the GitHub handle 😄 ).
> 
> I'll merge this after you accept my one suggestion.
> 
> Thanks again.

Thats done now. Thanks for looking at this PR. I will now be able to say "speaking as a dotnet contributor..." until I get rumbled.

**Review by BillWagner — APPROVED** [review]
Thanks again @CraigComeOnThisNameCantBeTaken 

I'll :shipit: now.

You should see the change on our live site in the next few days on our regular publishing cycle.

Thanks again for making your first contribution to docs! 🥂

**Pull request merged**